### PR TITLE
QA Service: Added detailed error message for Increment Counter Tx

### PR DESCRIPTION
**Pull request by bullet-tooth**
## Overview

---

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [x] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [x] Public API has Javadoc
- [x] Method preconditions are checked and documented in the Javadoc of the method
- [x] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

**Comment by MakarovS** [comment]
@bullet-tooth I think you wanted to merge that into new message format branch, not master?

**Comment by bullet-tooth** [comment]
@MakarovS correct!

File: exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/transactions/IncrementCounterTx.java [hunk 1]
```diff
@@ -53,7 +53,10 @@
    * @param counterId counter id, a hash of the counter name
    */
   public IncrementCounterTx(long seed, HashCode counterId) {
-    checkArgument(counterId.bits() == Hashing.DEFAULT_HASH_SIZE_BITS);
+    int size = counterId.bits();
+    checkArgument(size == DEFAULT_HASH_SIZE_BITS,
+        "Counter [%s] has %s bytes size but required size is %s",
```
Comment (dmitry-timofeev) [inline exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/transactions/IncrementCounterTx.java]: This message is not correct (bytes vs bits)

**Comment by bullet-tooth** [comment]
let's merge it to `new-message-format`. Merging it to `master` requires to create new branch + PR

**Comment by dmitry-timofeev** [comment]
OK, but please don't target anything non-message-format specific to that branch to avoid confusion.

**Pull request merged**